docs/site/tutorials/todo-list/todo-list-tutorial-relations.md

```diff
@@ -0,0 +1,141 @@
+permalink: /doc/en/lb4/todo-list-tutorial-relations.html
+summary: LoopBack 4 TodoList Application Tutorial - Add TodoList Repository
+---
+
+### Define the model relation
+
+We are going to add the model relation to indicate the relation that `TodoList`
+_hasMany_ `Todo` using the `lb4 relation` command.
+
+```sh
+$ lb4 relation
+? Please select the relation type hasMany
+? Please select source model TodoList
+? Please select target model Todo
+? Foreign key name to define on the target model todoListId
+? Source property name for the relation getter todos
+? Allow TodoList queries to include data from related Todo instances? Yes
+   create src/controllers/todo-list-todo.controller.ts
+
+Relation HasMany was created in src/
+```
+
+Now, we're going to add the relation for `Todo`. That is, `Todo` _belongsTo_
+`TodoList`.
+
+```sh
+$ lb4 relation
+? Please select the relation type belongsTo
+? Please select source model Todo
+? Please select target model TodoList
+? Source property name for the relation getter todoListId
+? Allow Todo queries to include data from related TodoList instances? Yes
+   create src/controllers/todo-todo-list.controller.ts
+
+Relation BelongsTo was created in src/
+```
+
+### Behind the scene
+
+If you want to understand the code changes introduced from the relation
+generator command, read on the details in this section; otherwise, you are ready
+to move to the next step to create the controller.
+
+#### Relation decorators
+
+When specifying the `hasMany` relation, it adds `@hasMany()` decorator defines
+this property. As the decorator's name suggests, `@hasMany()` informs LoopBack 4
+that a todo list can have many todo items.
+
+```ts
+export class TodoList extends Entity {
+  // ...properties defined by the CLI...
+
+  @hasMany(() => Todo)
+  todos?: Todo[];
+
+  // ...constructor def...
+}
+```
+
+Similarly for the `belongsTo` relation:
+
+```ts
+export class Todo extends Entity {
+  // ...properties of the Todo model
+
+  @belongsTo(() => TodoList)
+  todoListId: number;
+
+  // ...constructor def...
+}
+```
+
+#### Inclusion of Related Models
+
+When running the `lb4 relation` command, the prompt below registers the
+inclusionResolver for this belongsTo relation for you.
+
+```sh
+? Allow Order queries to include data from related Customer instances? (Y/n)
+```
+
+It defaults to Yes. Make sure to choose ‘yes’ if you’d like to use inclusion and
+your model is traversable. In the example of getting the related `Todo` objects
+for each `TodoList`, it registers the inclusion resolver that comes with the
+[`HasManyRepositoryFactory`](https://loopback.io/doc/en/lb4/apidocs.repository.hasmanyrepository.html).
+
+Let's take a closer look at the `TodoListRepository`.
+{% include code-caption.html content="src/repositories/todo-list.repository.ts" %}
+
+```ts
+this.todos = this.createHasManyRepositoryFactoryFor(
+  'todos',
+  todoRepositoryGetter,
+);
+this.registerInclusionResolver('todos', this.todos.inclusionResolver);
```

Comment:
Suggest:
add a comment 
// this line enables inclusion for this relation

Reply:
added it for both hasMany and belongsTo code snippet